core/src/main/scala/cats/data/NonEmptyVector.scala

```diff
@@ -329,8 +329,15 @@ final class NonEmptyVector[+A] private (val toVector: Vector[A])
     NonEmptySet.of(head, tail: _*)
 }
 
+sealed abstract private[data] class NonEmptyVectorInstances0 {
```

Comment:
we could move this below `NonEmptyVectorInstances` like it's done in `NonEmptyListInstances0`. It's pretty conventional across the board and makes the read top to bottom easier.